sources/api/prairiedog/src/bootconfig.rs

```diff
@@ -142,7 +142,8 @@ fn parse_value(input: &str) -> Result<BootConfigValue> {
     } else {
         input
     };
-    s.try_into().context(error::ParseBootConfigValueSnafu)
+    s.parse::<BootConfigValue>()
+        .context(error::ParseBootConfigValueSnafu)
```

Comment:
Requiring explicit types in places where they weren't required before doesn't really show this change in the best light.

It'd be better to use type inference where possible, e.g.
```
    s.parse::<>()
        .context(error::ParseBootConfigValueSnafu)
```